### PR TITLE
Added check for tika jar file, updated accompanying test

### PR DESCRIPTION
**Pull request by stephenfriedman**
Per this issue: https://github.com/google/go-tika/issues/17

Fixes the silent missing jar file error. Added accompanying test.

### CODE REVIEW

File: tika/server_test.go [hunk 1]
```diff
@@ -113,6 +113,10 @@ func TestStartError(t *testing.T) {
 	if err := s.Start(ctx); err == nil {
 		t.Fatalf("s.Start got no error, want error")
 	}
+	s.jar = "nonexistentFile.jar"
+	if err := s.Start(ctx); err == nil {
+		t.Fatalf("s.Start got no error, want error for missing Jar file")
+	}
```
Comment (tbpg) [inline tika/server_test.go]: Please add this to `TestStartError` (which may need to be updated to point to real files).

File: tika/server.go [hunk 2]
```diff
@@ -128,6 +128,10 @@ var command = exec.Command
 // Server. Start will wait for the server to be available or until ctx is
 // cancelled.
 func (s *Server) Start(ctx context.Context) error {
+	if _, err := os.Stat(s.jar); os.IsNotExist(err) {
+		return err
+	}
+	cmd := command("java", append([]string{"-jar", s.jar, "-p", s.port}, s.child.args()...)...)
```
Comment (tbpg) [inline tika/server.go]: Delete.

File: tika/server.go [hunk 3]
```diff
@@ -128,6 +128,10 @@ var command = exec.Command
 // Server. Start will wait for the server to be available or until ctx is
 // cancelled.
 func (s *Server) Start(ctx context.Context) error {
+	if _, err := os.Stat(s.jar); os.IsNotExist(err) {
+		return err
+	}
```
Comment (tbpg) [inline tika/server.go]: Should we do this in `NewServer`? Could see doing it in either function.

File: tika/server_test.go (hunk 1 above)
Comment (stephenfriedman) [reply tika/server_test.go]: This is in `TestStartError`. I am unsure of what your are asking.

File: tika/server.go (hunk 2 above)
Comment (stephenfriedman) [reply tika/server.go]: woops

File: tika/server.go (hunk 3 above)
Comment (stephenfriedman) [reply tika/server.go]: I agree, could be in either. Pathing of where things are called from could come into play, which is why I put it here and not in NewServer.

File: tika/server_test.go (hunk 1 above)
Comment (tbpg) [reply tika/server_test.go]: Ah. You're totally right, my bad.

File: tika/server.go (hunk 3 above)
Comment (tbpg) [reply tika/server.go]: Sounds good to me.

File: tika/server.go (hunk 2 above)
Comment (stephenfriedman) [reply tika/server.go]: woops. Repushed.

**Pull request merged**